docs/2.0. APIs.md

```diff
@@ -28,7 +28,13 @@ All NMOS APIs MUST use a path in the following format. Other HTTP resources MAY
 http(s)://<ip address or hostname>:<port>/x-nmos/<api type>/<api version>/
 ```
 
-At each level of the API from the base resource upwards, the response SHOULD include a JSON format array of the child resources available from that point.
+Nodes MAY extend the path for the Audio Channel Mapping API with a suffix. A Node implementation might choose to extend the path so that Devices on the Node have separate Audio Channel Map APIs without having to use a different port for each Device's API. Such a path SHALL be formatted as
+
+```
+http(s)://<ip address or hostname>:<port>/x-nmos/<api type>/<api version>/<device selector>/
```

Comment:
I used the word "selector" because "id" might imply the Device's UUID, which seems like an unnecessary restriction.

Do we need to constrain the form of the selector? In particular, would slashes within the selector present a problem for control systems?

Reply:
The variation should not matter - the controller would use the href in the control array as is.  If it does contain a suffix then that would be used as the base path for IS-08 methods.